common-module/src/main/java/domain/model/Category.java

```diff
@@ -0,0 +1,38 @@
+package domain.model;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import org.hibernate.annotations.CreationTimestamp;
+import domain.status.Exercise;
+
+@Entity
+public class Category {
+
+  @Id
+  @GeneratedValue
+  private long id;
+
+  @OneToOne(fetch = FetchType.LAZY)
+  private User user;
+
+  @Enumerated(value = EnumType.STRING)
+  private Exercise name;
+
+  @Column(nullable = false, updatable = false)
+  @CreationTimestamp
+  private LocalDateTime createdAt;
+  private LocalDateTime updatedAt;
+
+  @OneToMany(fetch = FetchType.LAZY)
+  private List<Board> members = new ArrayList<>();
```

Comment:
```OneToMany```는 ```mappedBy```를 설정해줘야 합니다.
변수 네이밍도 ```boards``` 가 더 괜찮을 것 같네요.
```suggestion
  @OneToMany(fetch = FetchType.LAZY, mappedBy = "category")
  private List<Board> boards = new ArrayList<>();
```